pyfftw/interfaces/_utils.py

```diff
@@ -97,24 +101,25 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
         # take the transform of the input array! (in general, we have
         # to assume that the input array will be destroyed during 
         # planning).
-        a_copy = a.copy()
+        if overwrite_input:    # Note: reload_after_transform is not True...
+            args = (a, s, axes) + args
+            a_copy = a.copy()  # ... hence this is not an additional copy
+        else:
+            args = (numpy.empty_like(a), s, axes) + args
```

Comment:
So if you call an `FFTW_object` with an incorrectly strided array, it will execute another plan?  (clobbering the original array should be easy to test for.)

I will think about the simplest logic that allows one to work with the original array if it is indeed strided.